packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -857,7 +857,7 @@ export class LocalFluidDataStoreContextBase extends FluidDataStoreContext {
         // Add GC details to the summary.
         const gcDetails: IGarbageCollectionSummaryDetails = {
             usedRoutes: this.summarizerNode.usedRoutes,
-            gcData: summarizeResult.gcData,
+            gcData: this.summarizerNode.gcData,
         };
```

Comment:
this seems a bit odd. will this gcData ever even be populated at attach time

Reply:
No it won't, this will just be undefined. We could just initial GC here but I don't think its worth. I can probably just remove adding this here.

Reply:
that's what i was thinking. i'd rather not set the field if it will always be undefined. this there back compat concerns here?

Reply:
There should not be any back compat concerns. If this does not exist and GC is enabled, we regenerate GC data.